.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -68,6 +68,13 @@ body:
       Please copy-paste the actual ESLint output. You can use Markdown in this field.
   validations:
     required: true
+- type: input
+    attributes:
+      label: Link to Minimal Reproducible Example
+      description: 'Link to a [playground](https://eslint.org/play), [StackBlitz](https://stackblitz.com), or GitHub repo with a minimal reproduction of the problem. **A minimal reproduction is required** so that others can help debug your issue. If a report is vague (e.g. just a generic error message) and has no reproduction, it may be auto-closed.'
+      placeholder: 'https://stackblitz.com/abcd1234'
+    validations:
+      required: true
```

Comment:
```suggestion
- type: input
  attributes:
    label: Link to Minimal Reproducible Example
    description: 'Link to a [playground](https://eslint.org/play), [StackBlitz](https://stackblitz.com), or GitHub repo with a minimal reproduction of the problem. **A minimal reproduction is required** so that others can help debug your issue. If a report is vague (e.g. just a generic error message) and has no reproduction, it may be auto-closed.'
    placeholder: 'https://stackblitz.com/abcd1234'
  validations:
    required: true
```